Packs/SafeBreach/Integrations/SafeBreach_v2/SafeBreach_v2_test.py

```diff
@@ -146,6 +147,30 @@ def test_get_indicators(requests_mock, mocker):
     assert res[0]['type'] == 'File'
 
 
+def test_get_indicators_exception(requests_mock, mocker):
+    mocker.patch.object(demisto, 'args', return_value={'limit': '10'})
+    mocker.patch.object(demisto, 'results')
+    mocker.patch.object(demisto, 'params', return_value={'url': MOCK_URL})
+
+    for insight_id in [5, 6, 8, 9, 13, 14, 17]:
+        requests_mock.get(f'{MOCK_URL}/api/data/v1/accounts/{MOCK_ACCOUNT_ID}/insights/{insight_id}/remediation',
+                          json=REMEDATION_DATA_LIST_UNVAILD_STRING)
+
+    requests_mock.get(
+        f'{MOCK_URL}/api/data/v1/accounts/{MOCK_ACCOUNT_ID}/insights?type=actionBased',
+        json=GET_INSIGHTS_LIST)
+    requests_mock.get(
+        f'{MOCK_URL}/api/config/v1/accounts/{MOCK_ACCOUNT_ID}/nodes?details=true&deleted=true&assets=true',
+        json=NODES)
+    insight_category = ['Endpoint', 'Web']
+    insight_data_type = ['Hash', 'Domain']
+    hash_to_search = '109c702578b261d0eda01506625423f5a2b8cc107b0d8dfad84d39fb02bfa5cb'
```

Comment:
Are you sure it's not a secret? if not,  please add it to the .secrets-ignore file.
else, you can change it like the suggestion:
```suggestion
    hash_to_search = '0000000000000000000000000000000000000000000000000000000000000000'
```